src/components/date-input/template.njk

```diff
@@ -52,16 +52,25 @@
   }) | indent(2) | trim }}
 {% endif %}
   <div class="govuk-date-input {%- if params.classes %} {{ params.classes }}{% endif %}"
-    {%- for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}
-    {%- if params.id %} id="{{ params.id }}"{% endif %}>
```

Comment:
I'm not sure this ID was doing anything, since it's not focusable? So potentially we should remove this regardless?